src/track-by-tracking-number/types/public.ts

```diff
@@ -1 +1,158 @@
-export {};
+export type Params = { trackingNumber: string; carrierCode: string };
+
+export type Response = TrackByTrackingNumberResult;
+
+/**
+ * The Tracking information and events associated with a label
+ * @see https://www.shipengine.com/docs/tracking/track-by-label-id/
+ */
+export interface TrackByTrackingNumberResult {
```

Comment:
I think we only want to export the `Params` and `Result`

Reply:
OOOH i have been doing response too...

Reply:
Ill fix mine